src/main/java/com/buuz135/industrial/block/resourceproduction/tile/FluidPlacerTile.java

```diff
@@ -68,7 +68,7 @@ public WorkAction work() {
                     tank.drainForced(FluidAttributes.BUCKET_VOLUME, IFluidHandler.FluidAction.EXECUTE);
                     increasePointer();
                     return new WorkAction(1, getPowerPerOperation);
-                } else if (world.isAirBlock(pointed)) {
+                } else if (world.isAirBlock(pointed) || !world.getFluidState(getPointedBlockPos()).isSource()) {
```

Comment:
You can change this too

Reply:
Yup, was just about to push that as well!